modules/rostests/apitests/ntdll/CMakeLists.txt

```diff
@@ -56,7 +56,6 @@ list(APPEND SOURCE
     RtlGetLongestNtPathLength.c
     RtlHandle.c
     RtlImageRvaToVa.c
-    RtlInitializeBitMap.c
```

Comment:
What is wrong with this test?

Reply:
It's the redundant test that is already covered with RtlBitmap.c in that same apitest module.